### PR TITLE
Remove model requirement from hybrid search documentation

### LINKED ISSUE #7401: [DOC] Incorrect Hybrid Search Documentation
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [X] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
In the documentation of hybrid search it is mentioned this `Before using a hybrid query, you must set up a machine learning (ML) model`

ReF: https://opensearch.org/docs/latest/query-dsl/compound/hybrid/#:~:text=Before%20using%20a%20hybrid%20query%2C%20you%20must%20set%20up%20a%20machine%20learning%20(ML)%20model

This is not correct. Setting of model is not required as hybrid search works with normal vector also. Due to this there has been a lot of confusion from customers that hybrid search requires ML model. This needs to be fixed.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
NA

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Closes #7401 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@navneet1v Could you review this PR please?

File: _search-plugins/hybrid-search.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Introduced 2.11
 Hybrid search combines keyword and neural search to improve search relevance. To implement hybrid search, you need to set up a [search pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) that runs at search time. The search pipeline you'll configure intercepts search results at an intermediate stage and applies the [`normalization_processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/normalization-processor/) to them. The `normalization_processor` normalizes and combines the document scores from multiple query clauses, rescoring the documents according to the chosen normalization and combination techniques. 
 
 **PREREQUISITE**<br>
-Before using hybrid search, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model).
+To follow this example, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model).
```
Comment (navneet1v) [inline _search-plugins/hybrid-search.md:15]: can we mention that having a embedding model is not required if you already have the embeddings for the text

Comment (kolchfa-aws) [reply _search-plugins/hybrid-search.md]: In this case, they can just directly set up hybrid search, correct? I added that sentence.

Comment (navneet1v) [reply _search-plugins/hybrid-search.md]: no actually thats not the case. What we need to tell them is if you already have the embeddings directly ingest the embeddings and skip the ingest pipeline creation step.

Comment (kolchfa-aws) [reply _search-plugins/hybrid-search.md]: Sure. I revised the sentence.

File: _search-plugins/hybrid-search.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Introduced 2.11
...
 
 **PREREQUISITE**<br>
-Before using hybrid search, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model).
+To follow this example, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model). If you already generated text embeddings, ingest the embeddings into an index and skip to [Step 4](#step-4-configure-a-search-pipeline).
```
Comment (natebower) [inline _search-plugins/hybrid-search.md:15]:
```suggestion
To follow this example, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model). If you have already generated text embeddings, ingest the embeddings into an index and skip to [Step 4](#step-4-configure-a-search-pipeline).
```

**Review by natebower — APPROVED** [review]
LGTM with a minor addition 😄

**Pull request merged**